indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -130,19 +130,21 @@ function EditableListDisplay({
         key: 'state',
         text: Translate.string('Status'),
         options: [
-          {value: 'not_submitted', text: Translate.string('Not submitted')},
-          {value: 'ready_for_review', text: Translate.string('Ready for review')},
-          {value: 'accepted', text: Translate.string('Accepted'), exclusive: true},
-          {value: 'rejected', text: Translate.string('Rejected'), exclusive: true},
+          {value: 'not_submitted', text: Translate.string('Not submitted'), color: 'black'},
```

Comment:
I'm not sure I agree with having black as the color for "not submitted", since we're repeating colors and not matching with what's displayed on the list.
Could you try to use the default label color here, to make it look something like this?
<img width="435" alt="image" src="https://user-images.githubusercontent.com/27357203/219851058-08079999-6a2c-472b-bffe-852e58b4027a.png">